backend/mora/service/detail_reading.py

```diff
@@ -642,11 +599,45 @@ def as_values(vs):
                     cache[v] = None
 
     # fetch and convert each object once, rather than multiple times
+    #
+
+    # handle cross-function links first, the only instance being this
+    # detail referring to an address
+    address_functions = collections.OrderedDict(
```

Comment:
I'd write this further up if I could; the support for handling string value in `as_values()` is now unused dead, and may safely be removed.